### PR TITLE
Fix/change jurisdiction notification url & add Terms of Service

### PR DESCRIPTION
**Pull request by phattya**
# Pull Request Template

## Issue Overview
The notification URL was linked within the code base, rather than pulling the value from the db. Seed data has been updated with the correct GovDelivery link and changed code that references that URL.

TODO: Need to update ListingsCombined & rebase

Link is: https://public.govdelivery.com/accounts/CAMTC/signup/36832

This PR addresses #issue

- [x] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## How Can This Be Tested/Reviewed?
Reseed data using yarn db:reseed`, then `yarn install` & `yarn dev` on the public site and verify that the link on the Listings page links to the correct govDelivery site.

go to `http://localhost:3100/jurisdictions` and see that the `notificationsSignUpURL` URL is now https://public.govdelivery.com/accounts/CAMTC/signup/36832

image

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [ ] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: backend/core/src/seeder/seeds/jurisdictions.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export const activeJurisdictions: JurisdictionCreateDto[] = [
     multiselectQuestions: [],
     languages: [Language.en],
     publicUrl: "",
-    notificationsSignUpURL: "https://public.govdelivery.com/accounts/CAALAME/signup/29652",
+    notificationsSignUpURL: "https://public.govdelivery.com/accounts/CAMTC/signup/36832",
```
Comment (lisunshiny) [inline backend/core/src/seeder/seeds/jurisdictions.ts:12]: (related to the magic strings vs. env variables suggestion in an earlier PR) we need to change this in `testHelpers.ts` as well

**Review by lisunshiny — APPROVED** [review]

**Comment by phattya** [comment]
@lisunshiny I finally got to review this and make sure it builds. Can you build & run this & double check my work? Seems like I had to do more than I would expect to add this in.

Also, I'm not sure if this is related, but on localhost I don't see the sign-up section of the Landing page ... so slightly worried I'm introducing a bug.

What I see on localhost:
image

What it should look like: 
https://screenshot.googleplex.com/3CPvohqbQpPpQ8p.png

File: sites/public/src/components/listings/search/ListingsSearchCombined.tsx [hunk 2]
```diff
@@ -11,6 +11,7 @@ import {
   t,
 } from "@bloom-housing/doorway-ui-components"
 import { FormOption, ListingsSearchModal } from "./ListingsSearchModal"
+import { jurisdiction } from "@bloom-housing/shared-helpers/__tests__/testHelpers"
```
Comment (chriscasto) [inline sites/public/src/components/listings/search/ListingsSearchCombined.tsx:14]: I realize that we're down to the wire with getting these changes finalized, but it makes me a bit uneasy to use a dummy jurisdiction in testHelpers in place of the actual value from the database.  Can we just call `fetchJurisdictionByName` in `getServerSideProps` and pass it in that way?

**Review by chriscasto — CHANGES_REQUESTED** [review]

File: sites/public/src/components/listings/search/ListingsSearchCombined.tsx (hunk 2 above)
Comment (isaac-looker) [reply sites/public/src/components/listings/search/ListingsSearchCombined.tsx:14]: I'm getting `  message: 'Not Found',` from the backend when calling

```
  const jurisdiction = fetchJurisdictionByName(        
    runtimeConfig.getBackendApiBase(),                                                                                                                                            
    runtimeConfig.getJurisdictionName()                                                                                                                                                             
  ) 
```
in dev. When we reseed in dev, we're not adding the "Bay Area" jurisdiction, that I can see.

Comment (isaac-looker) [reply sites/public/src/components/listings/search/ListingsSearchCombined.tsx:14]: nm. I see it exists but my public site isn't sending it at the moment.

**Review by chriscasto — APPROVED** [review]

**Comment by phattya** [comment]
Awesome! Thanks for spending time on that @isaac-looker!!! I've been busy with prepping for this week's training & hand off. I really appreciate it.

**Pull request merged**